### ISSUE TITLE
Some experiment detail?

### ISSUE THREAD

**Issue by carycode4fun** [sebgao/LIP#3]
Hi,I have run your code on my PC after I read your paper and follow the instruction on your paper include the training configuration, but I the experiment results of MSCOCO 2017 is quite different.
For example,I use the backbone LIP-ResNet-50 and the pretrained model you provide, the result is AP: 32.3, AP_50: 52.4, AP_75: 34.5, AP_s: 18.9, AP_m: 35.4, AP_l: 40.5,the result is lower than you almost 7%,So Can you tell me where I am wrong or Some experiment detail information?

**Comment by sebgao** [comment]
Hi. Are you using mmdetection and default training config files? By the way, how many gpus are you using? It's most likely that the learning rate does not match the number of gpus in mmdetection.

**Comment by carycode4fun** [comment]
Thanks for your reply,I forgot to tell you that I only have 2 1080Ti, and I run twice also conduct on mmdetection, lr is set according to the batch size first time and the lr is default 0.02 for the second time,batch size is 4 for both time

**Comment by sebgao** [comment]
You can refer to https://github.com/open-mmlab/mmdetection/issues/25 and https://github.com/open-mmlab/mmdetection/issues/686 for the learning rate setting. In a word, `lr=0.02` is for the total batch size 2*8=16. If you scale the total batch size, you should rescale `lr` linearly correspondingly.

**Comment by carycode4fun** [comment]
I actually do scale the lr according to the batch size, It's also lower than your performance almost 10%,I am so confuse, by the way,your paper said your training time is 2x compare with the default mmdetection setting, is that you training 24 epoch total?

**Comment by sebgao** [comment]
Yes, it's totally 24 epoch. I'm also confused that the performance should not be that low (32.3 AP you reported, 38.1 AP I got) even with 1x lr schedule. One thing to refer, I got 21.4 AP at the first epoch when training Faster R-CNN FPN w LIP-ResNet-50.

By the way, can you reproduce `Faster R-CNN R-50-FPN` in https://github.com/open-mmlab/mmdetection/blob/master/docs/MODEL_ZOO.md?

**Comment by carycode4fun** [comment]
That's huge difference!! I do the same but I get 7.4 AP.Why are your results so high??

**Comment by sebgao** [comment]
I believe it should not be that low. The plain R-50-FPN also got 19.3 AP at the first epoch. Are you sure the pretrained model is loaded?

**Comment by carycode4fun** [comment]
I download the pretrained model from your page,then modify the mmdetection config file 
`model = dict(
    type='FasterRCNN',
    pretrained = '/mnt/disk1/mmdet_work_dirs/LIP_faster_rcnn/pretrained_model/lip_resnet-50.pth',
    backbone=dict(
        type='ResNet_Lip',
        depth=50,
        ),`

**Comment by sebgao** [comment]
Your config file is correct. Have you frozen BNs' statistics, `conv1` and `res_layers[0]` in your `ResNet_Lip`?

**Comment by carycode4fun** [comment]
No,Maybe it's reason.Can you provide a example?

**Comment by sebgao** [comment]
Cool! Finally found the reason. You can refer to `_freeze_stages` and `train` in https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/backbones/resnet.py and `frozen_stages` in https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn_r50_fpn_1x.py.

But I recommend that you first copy `resnet.py` in mmdetection and modify it to the LIP one because the backbone in mmdetection need careful and subtle treating. Hope it works. Looking forward to your feedback.

**Comment by carycode4fun** [comment]
OK,I will try,I will tell you if it's work.

**Comment by sebgao** [comment]
> OK,I will try,I will tell you if it's work.

How is the experiment going?

**Comment by carycode4fun** [comment]
Sorry,I have some other experiments to do,I will verify this problem when other experiment is completed.

**Comment by sebgao** [comment]
Given that this issue seems to go out of date, I'm about to close it. If you still have concerns with this problem, please feel free to reopen it. I'm still looking forward to your results. Thank you.
